src/v2/Apps/Auction/Components/Form/helpers.tsx

```diff
@@ -42,8 +40,8 @@ export const getSelectedBid = ({
 }
 
 export const determineDisplayRequirements = (
-  bidder: BidForm_saleArtwork["sale"]["registrationStatus"],
-  me: BidForm_me
+  bidder: object,
```

Comment:
Curious to learn more here – why reduce the strictness of the type?

Reply:
We only check the presence of the object so we simply don't need very strict type checks here. We could promote it up to a more concrete type when we start using more fields/functions.